### PR TITLE
networkd: Add comment for setting_mtu #7460

### PR DESCRIPTION
**Pull request by ssahani**
Follow up #7460

### CODE REVIEW

File: src/network/networkd-link.c [hunk 1]
```diff
@@ -3108,6 +3108,9 @@ static int link_carrier_lost(Link *link) {
 
         assert(link);
 
+        /* Some devices reset itself while setting the MTU. This trigers the DHCP client fall into a LOOP.
```
Comment (yuwata) [inline src/network/networkd-link.c]: trigers -> triggers

Comment (keszybz) [reply src/network/networkd-link.c]: Also you cannot use "trigger" with "into".

Comment (ssahani) [reply src/network/networkd-link.c]: what should be the correct way then ?

**Comment by keszybz** [comment]
For example "This causes the client to fall into a loop."

**Comment by yuwata** [comment]
Hmm... The second sentence is not easy for me. But anyway, this is just a comment. I do not disturb this anymore if @keszybz approve it.

**Comment by keszybz** [comment]
OK, let's merge this, and somebody will fix the grammar later.

**Pull request merged**